src/js/api.ts

```diff
@@ -557,12 +557,28 @@ export class PyodideAPI {
     );
 
     // sync native ==> browser
-    await new Promise((resolve, _) => Module.FS.syncfs(true, resolve));
+    await new Promise<void>((resolve, reject) => {
+      Module.FS.syncfs(true, (err: any) => {
+        if (err) {
+          reject(err);
+        } else {
+          resolve();
+        }
+      });
+    });
```

Comment:
Can we make helper functions at top level like:
```js
function syncfs(direction: bool): Promise<void> {
   return new Promise((resolve, reject) => {
      Module.FS.syncfs(true, (err: any) => {
        if (err) {
          reject(err);
        } else {
          resolve();
        }
      });
    });
}

async function syncBrowserToNative() {
    return await syncfs(true);
}

async function syncNativeToBrowser() {
    return await syncfs(true);
}
```

Reply:
FWIWI that was my idea for the port but I didn't want to create branched code ... also you forgot to use `direction` in that example, it's always `true` and I believe that's not what you meant to write.

Reply:
never mind ... it was already addressed!